packages/vite-plugin-docs/docs/getting-started/solid/03-dev-content-script.md

```diff
@@ -0,0 +1,68 @@
+---
+id: content-script-hmr
+title: Content script HMR
+description: Developing a Solid content script with Vite and CRXJS
+tags:
+  - Content script
+  - Solid
+# could point this to a TypeScript manifest guide
+pagination_next: concepts/manifest
+---
+
+# Solid and Content Scripts
+
+Make sure that your extension is [loaded in the browser](dev-basics) and that
+you've started Vite. Navigate to `https://www.google.com` and scroll to the
+bottom of the page.
+
+There's our familiar Vite-Solid Hello World!
+
+<!-- TODO: add screenshot of raw content script -->
+
+## Leaking CSS styles
+
+Notice how the counter button doesn't look like a button. That's because
+Google's styles affect our content script elements. The same goes the other way:
+our styles change Google's styles.
```

Comment:
The Solid example doesn't have a button, so we need to do one of these:

- add a button/counter
- add a different way to demonstrate HMR on a content script

Feel free to do what you want with this section! We don't have to demonstrate leaking CSS styles here; the point is to demonstrate HMR on content scripts.

Reply:
I think we could not stray too far from the React guide by creating a button. https://github.com/crxjs/chrome-extension-tools/pull/476/commits/f1523f51b365edd2a8d0c4d27794f04f897cb271

Reply:
@ryoid I love it, let's ship it 🚀